provider/database.py

```diff
@@ -83,33 +83,37 @@ def createCanary(self):
         maxRetries = 3
         retryCount = 0
 
+        instance = os.getenv('INSTANCE', 'messageHubTrigger-0')
```

Comment:
Feels like the assignments to `instance` and `canaryId` should occur outside of this function. Otherwise, you are unnecessarily redefining them every time `createCanary()` is invoked.